core/src/main/java/com/hcl/ecomm/core/models/package-info.java

```diff
@@ -14,6 +14,6 @@
  *  limitations under the License.
  */
 @Version("1.0")
-package com.hcl.ecomm.core;
```

Comment:
@DeepaliAgrwl Why do we have this merge and file ?

core/src/main/java/com/hcl/ecomm/core/package-info.java → core/src/main/java/com/hcl/ecomm/core/models/package-info.java

Reply:
@rishabh-behl we don't have any file at this location 
 core/src/main/java/com/hcl/ecomm/core/package-info.java 

Also I have removed the file this location core/src/main/java/com/hcl/ecomm/core/models/package-info.java

So, Please check and let us know if the issue persists